ParquetCSVImporter/ClassMaps/BlockClassMap.cs

```diff
@@ -0,0 +1,25 @@
+﻿using CsvHelper.Configuration;
+
+namespace ParquetCSVImporter.ClassMaps
+{
+    /// <summary>
+    /// Maps the values in a <see cref="T:ParquetCSVImporter.Shims.BlockShim"/> to records that CSVHelper recognizes.
+    /// </summary>
+    public class BlockClassMap : ClassMap<BlockShim>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:ParquetCSVImporter.ClassMaps.BlockClassMap"/> class.
+        /// </summary>
+        public BlockClassMap()
+        {
+            Map(m => m.ID).Index(0);
```

Comment:
I don't have an informed opinion about the following, and I'm asking in case it wasn't considered;

What made you favor Index instead of Name for fetching values? e.g. Map(m => m.ID).Name("Name").

Reply:
I think that you could also then do this;
Map(m => m.ID).Name(nameof(BlockShim.ID));

Reply:
So, the reason I favoured indexes is because I wanted to be sure what order the names showed up in.  For editing purposes, it's easiest for me if the ID comes first in each row, followed by the properties in the order I think about them.

I think I will keep this as is for now, but thank you for pointing this out!